### PR TITLE
add getters

### PR DESCRIPTION
**Pull request by danielnaumau**

### CODE REVIEW

File: src/main/java/com/traveltime/sdk/dto/responses/errors/IOError.java [hunk 1]
```diff
@@ -1,8 +1,10 @@
 package com.traveltime.sdk.dto.responses.errors;
 
 import lombok.AllArgsConstructor;
+import lombok.Getter;
 import lombok.NonNull;
 
+@Getter
```
Comment (mjanuszkiewicz-tt) [inline src/main/java/com/traveltime/sdk/dto/responses/errors/IOError.java:7]: I think here you also want `getMessage` to return `errorMessage`?

Comment (danielnaumau) [reply src/main/java/com/traveltime/sdk/dto/responses/errors/IOError.java:7]: what do you mean?

Comment (danielnaumau) [reply src/main/java/com/traveltime/sdk/dto/responses/errors/IOError.java:7]: getter will generate getErrorMessage

Comment (mjanuszkiewicz-tt) [reply src/main/java/com/traveltime/sdk/dto/responses/errors/IOError.java:7]: Yes, but I think the `getMessage` method that now returns `cause.getMessage` should actually return `errorMessage`. That way if you call `getMessage` on any error returned by the sdk you will get a message provided by us.
Currently `getMessage` will return a message from some exception from IOError and custom messages everywhere else.

Comment (danielnaumau) [reply src/main/java/com/traveltime/sdk/dto/responses/errors/IOError.java:7]: Okey, I changed it

**Review by mjanuszkiewicz-tt — APPROVED** [review]

**Pull request merged**